numpy/_globals.py

```diff
@@ -79,3 +80,35 @@ def __repr__(self):
 
 
 _NoValue = _NoValueType()
+
+
+class _NeverCopyType(object):
+    """Special `copy` keyword value.
+
+    The instance of this class may be used for many `copy` arguments to
+    functions to indicate that a copy must not be made and an error will
+    be raised when this cannot be guaranteed.
+
+    In the future the string "never" may replace this variable.
+    """
+    __instance = None
+    def __new__(cls):
+        # ensure that only one instance exists
+        if not cls.__instance:
```

Comment:
Hmm, I copied this from above and it seems to work `bool(np.never_copy)` does raise the error for me, so I am not sure I understand. But maybe this can be much simpler anyway, since the pickle stuff is not necessary.

Reply:
Ohhhh, sorry, that was silly.